### DISCUSSION TITLE
Shouldn't `userEvent` be only exported as a named export ?

### DISCUSSION THREAD

**Discussion by mlz11** [testing-library/user-event#1205] [Q&A] [answered]
Hey everyone 👋🏻

Could someone help me understand what's the point of exporting `userEvent` as a default export **and** as a named export ?

```
export { userEvent as default } from './setup';
export { userEvent } from './setup';
export type { UserEvent } from './setup/setup';
export type { keyboardKey } from './system/keyboard';
export type { pointerKey } from './system/pointer';
export { PointerEventsCheckLevel, type Options } from './options';
```

Using automatic import I get the following which triggers the following ESLint rule `eslintimport/no-named-as-default`

```
import userEvent from '@testing-library/user-event'; 
// ESLint expects 'import { userEvent } from '@testing-library/user-event';'
```

**Comment by mlz11** [comment, marked as answer]
Got the answer on Twitter ⬇️

<img width="616" alt="image" src="https://github.com/testing-library/user-event/assets/94069699/244173fc-a2f7-492c-b46e-14cbf2785057">

**Comment by ahayes91** [comment]
https://testing-library.com/docs/user-event/intro/ examples currently use the default export instead of named, probably needs a quickie PR to fix - https://github.com/testing-library/testing-library-docs/pull/1476
